docs/docs/troubleshooting/TROUBLESHOOTING.md

```diff
@@ -104,3 +104,16 @@ This guide contains general tips on how to investigate an application deployment
     There are two Terraform locks; one for the infrastructure and another for Terraform state. If you are still experiencing lock issues, change the directory to `./modules/tfstate` and retry the same command.
 
 
+??? tip "How do I deal when remote agents are offline after provision the Bamboo?"
+       
+    **Symptom**
+    
+    The remote agents are installed but remain offline after installation. 
+       
+    **Solution**
+    
+    1. Open the Bamboo application in the browser and login as `administrator`. 
```

Comment:
```suggestion
    1. Open the Bamboo application in the browser and log in as _Administrator_. 
```

The admin can be called something else but the administrator concept is something common.